tm2/pkg/crypto/keys/client/broadcast.go

```diff
@@ -87,6 +88,7 @@ func execBroadcast(cfg *BroadcastCfg, args []string, io commands.IO) error {
 		io.Println("GAS USED:  ", res.DeliverTx.GasUsed)
 		io.Println("HEIGHT:    ", res.Height)
 		io.Println("EVENTS:    ", string(res.DeliverTx.EncodeEvents()))
+		io.Println("TX HASH:   ", base64.StdEncoding.EncodeToString(res.Hash))
```

Comment:
This should be also printed in the other case, when the transaction's delivery failed (it's a good transaction, but failed when executing):

`if res.DeliverTx.IsErr() {...}`

Reply:
https://github.com/gnolang/gno/pull/2309/commits/9c3a0f694871ffa3625387ae10ac410d6bfebee0